### PR TITLE
docs(ui5-combobox): add storybook samples

### PR DESCRIPTION
**Pull request by MapTo0**

### CODE REVIEW

File: packages/playground/_stories/main/ComboBox/ComboBox.stories.ts [hunk 1]
```diff
@@ -10,124 +12,113 @@ import { DocsPage } from "../../../.storybook/docs";
...
+		?disabled="${ifDefined(args.disabled)}"
+		?readonly="${ifDefined(args.readonly)}"
+		?required="${ifDefined(args.required)}"
+		?loading="${ifDefined(args.loading)}"
+		filter="${ifDefined(args.filter)}"
+		value-state="${ifDefined(args.valueState)}"
+		accessible-name="${ifDefined(args.accessibleName)}"
+		accessible-name-ref="${ifDefined(args.accessibleNameRef)}"
+>
+		${unsafeHTML(args.default)}
```
Comment (Okiana) [inline packages/playground/_stories/main/ComboBox/ComboBox.stories.ts:48]: Could you please also add the valueStateMessage and icon slots here

**Comment by ivoplashkov** [comment]
https://user-images.githubusercontent.com/23635149/233021032-97527596-67b7-47d2-9e09-f5dc2571a295.mov

I noticed a strange behaviour, which I am not sure if it's related to your change or not.

**Comment by nnaydenow** [comment]
Can you change the title of the PR from `feat(..)..` to `docs(..)..`?

**Comment by MapTo0** [comment]
> 

Yes, I have noticed that too, I guess we need to disable the playground shortcuts for samples

image

**Review by Okiana — APPROVED** [review]

**Pull request merged**